src/modules/submit.js

```diff
@@ -0,0 +1,15 @@
+export default async function submitScore(url = '', data = {}) {
```

Comment:
:red_circle: [REQUIRED] As per [this requirement ](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md):arrow_down: You should use arrow functions instead of the function keyword.
![req](https://github.com/ivangonzalez224/Leaderboard/assets/26736582/af9dcc97-2146-464d-95aa-7b5debabff26)